Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1234,6 +1298,29 @@ def search_file_command(file):
     get_indicator_outputs(indicator_type, indicator_details, 'SHA256')
 
 
+def get_export_list_command(args):
+    data = {
+        'label': args.get('label'),
+        'panosFormatted': True,
+        'apiKey': ''
+    }
+    results = http_request(url_suffix='/export', method='POST', data=data,
+                           err_operation=f"Failed to fetch export list: {args.get('label')}")
+
+    indicators = []
+    for indicator_value in results.get('export_list'):
+        indicator_type = type_of_indicator(indicator_value)
+        indicators.append({
+            'Type': indicator_type,
+            'Value': indicator_value,
+        })
+
+    hr = tableToMarkdown(f"Export list {args.get('label')}", indicators, headers=['Type', 'Value'])
+
+    return_outputs(hr, {'AutoFocus.ExportListIndicator(val.Value == obj.Value && val.Type == obj.Type)': indicators},
```

Comment:
why under ExportListIndicator

Reply:
simply Indicator? I wanted to differentiate this from other indicators

Reply:
@avidan-H do we have a convention for this?

Reply:
No we don't. We should discuss if it makes sense to establish one. Also, shouldn't output entries for each of the indicator entities (different for each indicator type) be included in the outputs? https://github.com/demisto/content-docs/blob/master/docs/context-standards.md

Reply:
Well the command literally returns a list of indicators with no additional info like you can see on the context standards. 

You can see an output example here: https://docs.paloaltonetworks.com/autofocus/autofocus-api/perform-direct-searches/export-list#id1b0b954a-f74d-462d-805b-8916455bf7b0_id259466ba-307f-42b0-8e1d-f99067576c84

Also considering this is supposed to be "minerish" shouldn't be up to the standards set to other miners 
(type: *, value: *)?

Reply:
You can and need to add whatever you can get from the product and populate the relevant fields from the context standards.
like ip.address

Reply:
done

Reply:
done